WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -981,13 +979,12 @@ decisions = {
 			NOT = { religion_group = muslim }
 			OR = {
 				NOT = { religion_group = pagan_group }
-				religion = zun_pagan
-				religion = zun_pagan_reformed
 				AND = {
 					religion_group = pagan_group
-					realm_size = 80
+					realm_size = 20
```

Comment:
No, not by the sense of this decision. This is special support through the weight of your realm. Standard support is already possible through the buildings. Think of it like what different Chinese dynasties did, but couldn't do during periods of weakness.